NVorbis/README

```diff
@@ -0,0 +1,123 @@
+    * Added multi-stream support to VorbisWaveReader
+    * Added several info properties to VorbisWaveReader
+0.5.5 - 11/15/2012
+    * Added OpenTK support
+    * Added makefiles for Mono
+    * Added constructor to NAudioSupport for already opened Stream instances
+    * Made Ogg container logic thread-safe
+    * Cleanup & Optimization
+0.6.0 - 06/27/2013
+    * Remove ACache from project
+    * Make clipping optional (default: on)
+    * Set clipping to clamp at +/-.99999994 instead of +/-1 as a courtesy to those not outputing float samples
+    * Performance improvements for RingBuffer
+    * Performance improvements in ThreadSafeStream
+0.7.0 - 07/15/2013
+    * Rewrite Ogg Reader buffering logic to vastly improve memory usage
+    * Reorganize to more easily support other container types / work with other Ogg stream decoders
+    * Exposed Ogg reading types
+    * Performance improvements
+    * Bugfix: Don't throw on Vorbis streams that only have a single mode
+0.7.1 - 07/18/2013
+    * Removed use of generics & lambda from Huffman decoder
+    * Reduced computation by only processing samples that are required for output
+    * Bugfix: Seeking now works again
+    * Bugfix: OpenTKSupport.OggStream now only queues as many source buffers as it filled
+0.7.2 - 07/22/2013
+    * Bugfix: Floor 0 implementation is now correct
+    * Bugfix: Residue 0 & 1 implementation is now correct
+    * Bugfix: Seeking back less than the current buffer size no longer corrupts the read buffer
+0.7.3 - 08/02/2013
+    * Bugfix: Fix residue 1
+0.7.4 - 09/16/2013
+    * Bugfix: Some stereo files with sound on only one channel decoded noise on the silent channel.
+0.8.0test1 - 09/16/2013
+    * Rebuilt container reading infrastructure so it is properly decoupled from the decoder
+    * Added unit tests for the packet reader, Ogg container reader
+0.8.0 - 01/16/2014
+    * Minor bugfixing to test1
+    * Replaces 0.7.4 as recommended release
+0.8.1 - 02/24/2014
+    * Bugfix: Threading issues in Ogg reader
+    * Rewrite StreamReadBuffer (again) for simplicity and correctness
+    * Add "saved buffer" concept to StreamReadBuffer for multi-threaded performance
+    * Reduce Ogg reader memory usage
+    * Add locking for multi-threading support in Ogg reader
+    * Remove exception-based logic where possible for better performance
+0.8.2 - 03/19/2014
+    * BugFix: Don't try to copy StreamReadBuffer data when offset is bigger than source
+    * Add parameter change event to packet provider interface
+    * Refactored decoder setup so parameter change can be handled
+    * Rewrote packet peek / get logic for simplicity
+    * Refactored locking in OggPacketReader / OggContainerReader
+    * Improved Ogg end of stream handling
+    * Changed container seek API to use packets instead of indexes to reduce linked list traversals
+0.8.3 - 04/29/2014
+    * BugFix: Don't set the end of stream flag until the last packet of the page
+0.8.4 - 11/20/2014

```

Comment:
Is this version accurate as to the latest changes in this build?  Or can you provide an svn/git revision so that we know exactly what version of the code was checked in here.